src/include/btmem.h

```diff
@@ -821,13 +820,15 @@ struct __wt_ref {
     WT_PAGE *volatile home;        /* Reference page */
     volatile uint32_t pindex_hint; /* Reference page index hint */
 
-#define WT_REF_DISK 0        /* Page is on disk */
-#define WT_REF_DELETED 1     /* Page is on disk, but deleted */
-#define WT_REF_LOCKED 2      /* Page locked for exclusive access */
-#define WT_REF_MEM 3         /* Page is in cache and valid */
-#define WT_REF_READING 4     /* Page being read */
-#define WT_REF_SPLIT 5       /* Parent page split (WT_REF dead) */
-    volatile uint32_t state; /* Page state */
+#define WT_REF_DISK 0       /* Page is on disk */
+#define WT_REF_DELETED 1    /* Page is on disk, but deleted */
+#define WT_REF_LOCKED 2     /* Page locked for exclusive access */
+#define WT_REF_MEM 3        /* Page is in cache and valid */
+#define WT_REF_READING 4    /* Page being read */
+#define WT_REF_SPLIT 5      /* Parent page split (WT_REF dead) */
```

Comment:
What is different about all the `WT_REF_*` defines?

Reply:
Nothing, clang-format redid the comment formatting.

Reply:
Ah. I see now. One space...